recipes/stark/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "v0.1" %}
```

Comment:
Conda's convention is to not have versions starting with "v", https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#package-version. Please use "0.1" in the version field, and use "https://github.com/hnikaein/stark/archive/v{{version}}.tar.gz" for the url below.